portfolio/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -26,11 +27,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Gson gson = new Gson();
     if (userService.isUserLoggedIn()) {
       String logoutUrl = userService.createLogoutURL("/");
-      response.getWriter().println("["+gson.toJson(true)+","+gson.toJson(logoutUrl)+"]");
+      AuthResponse a = new AuthResponse(true, logoutUrl);
```

Comment:
Here and below - when passing literal parameters, adding a comment like /* parameterName= */ can make the code more readable. 

Here's a snippet from the Java Style Guide: 


In certain contexts literal arguments - such as 0, "", true and false, or null - can make it difficult for readers to know what a method will do. We generally discourage defining methods that take boolean parameters or otherwise expect users to pass in ambiguous literals. However, when you must call such a method you're encouraged to use the parameter name as an inline comment at the call site, so that readers don't need to look at the method declaration to understand the parameter's purpose.

Such a comment should be formatted in this way:

`booleanMethod(/* enableFoo= */ true);`

Reply:
I like that, thank you!